felix/bpf/conntrack/bpf_scanner.go

```diff
@@ -40,88 +35,45 @@ const (
 	BPFLogLevelNone  BPFLogLevel = "no_log"
 )
 
-var (
-	registerOnce sync.Once
-
-	gaugeVecConntrackEntries = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "felix_bpf_conntrack_entries_seen",
-		Help: "Number of entries seen in the conntrack table at the last GC sweep, grouped by type.",
-	}, []string{"type"})
-	counterVecConntrackEntriesDeleted = prometheus.NewCounterVec(prometheus.CounterOpts{
-		Name: "felix_bpf_conntrack_entries_deleted",
-		Help: "Cumulative number of entries deleted from the conntrack table, grouped by type.",
-	}, []string{"type"})
-	summaryCleanerExecTime = prometheus.NewSummary(prometheus.SummaryOpts{
-		Name: "felix_bpf_conntrack_cleaner_seconds",
-		Help: "Time taken to run the conntrack cleaner BPF program.",
-	})
-)
-
-func registerConntrackMetrics() {
-	registerOnce.Do(func() {
-		prometheus.MustRegister(
-			gaugeVecConntrackEntries,
-			counterVecConntrackEntriesDeleted,
-			summaryCleanerExecTime,
-		)
-	})
-}
-
-// BPFProgLivenessScanner is a scanner that uses a BPF program to scan the
+// BPFProgCleaner uses a BPF program to scan the
 // conntrack table for expired entries.  The BPF program does the entry
```

Comment:
Out of date: no longer scans the conntrack table, it scans the cleanup table only